packages/runtime/container-runtime/src/gcSweepReadyUsageDetection.ts

```diff
@@ -130,7 +148,7 @@ export class SweepReadyUsageDetectionHandler {
 
         const error = new SweepReadyUsageError(
             "SweepReady object used in Non-Summarizer Client",
```

Comment:
The error message should be renamed to something that an end-user / partners can understand. I don't think they care about summarizer or non-summarizer clients. Maybe it should just say "object used after garbage collected"?
We can add another field in the errorDetails to say this was an interactive client.

Reply:
Sure, good feedback.  clientType should be set anyway on all these logs